lib/components/form/call-taker/mode-dropdown.tsx

```diff
@@ -0,0 +1,58 @@
+import { connect } from 'react-redux'
+import { TransportMode } from '@opentripplanner/types'
+import React, { useState } from 'react'
+
+import { defaultDropdownConfig } from '../../../util/call-taker'
+import { getModuleConfig, Modules } from '../../../util/config'
+
+type DropdownOption = {
+  combination: TransportMode[]
+  label: string
+}
+
+type Props = {
+  modeDropdownOptions: DropdownOption[]
+  onChangeModes: (combination: TransportMode[]) => void
+}
+
+/**
+ * Dropdown selector for the Call Taker form that allows quick selection of the
+ * full set of exclusive modes and access modes + transit. This will also
+ * automatically apply any companies associated with the mode option to the
+ * query params (e.g., Uber for CAR_HAIL or one of the various bike/scooter
+ * rental companies).
+ */
+function ModeDropdown({ modeDropdownOptions, onChangeModes }: Props) {
+  const [selectedMode, setSelectedMode] = useState(modeDropdownOptions[0].label)
+
+  const onChange = React.useCallback(
+    (e: React.FocusEvent<HTMLSelectElement>) => {
+      setSelectedMode(e.target.value)
+      const newModes = modeDropdownOptions.find(
+        (mdo) => mdo.label === e.target.value
+      )?.combination
+      onChangeModes(newModes || [])
+    },
+    []
+  )
+
+  return (
+    <select onBlur={onChange} onChange={onChange} value={selectedMode}>
```

Comment:
Note for another PR: not sure if it is a config issue, but the "Rail only" option seems to be too strict, returning no itineraries whereas the "transit" option produces a Walk-MAX itinerary.

Reply:
Probably just bad config.